satellite/gracefulexit/endpoint.go

```diff
@@ -573,6 +574,18 @@ func (endpoint *Endpoint) handleSucceeded(ctx context.Context, pending *pendingM
 
 	pending.delete(originalPieceID)
 
+	deleteMsg := &pb.SatelliteMessage{
+		Message: &pb.SatelliteMessage_DeletePiece{
+			DeletePiece: &pb.DeletePiece{
+				OriginalPieceId: originalPieceID,
+			},
+		},
+	}
+	err = stream.Send(deleteMsg)
```

Comment:
What do you think about commenting this out and putting a `TODO` to put it back in after the pointer updates are merged?

Reply:
This is actually already happening on master.  I just moved it from
```
		case *pb.StorageNodeMessage_Succeeded:
			err = endpoint.handleSucceeded(ctx, pending, nodeID, m)
			if err != nil {
				if ErrInvalidArgument.Has(err) {
					return rpcstatus.Error(rpcstatus.InvalidArgument, err.Error())
				}
				return rpcstatus.Error(rpcstatus.Internal, err.Error())
			}
			deleteMsg := &pb.SatelliteMessage{
				Message: &pb.SatelliteMessage_DeletePiece{
					DeletePiece: &pb.DeletePiece{
						OriginalPieceId: m.Succeeded.OriginalPieceId,
					},
				},
			}
			err = stream.Send(deleteMsg)
			if err != nil {
				return rpcstatus.Error(rpcstatus.Internal, Error.Wrap(err).Error())
			}
```
to `handleSucceeded`

Reply:
what was the reason for moving this part inside handleSucceeded?